shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Chain.hs

```diff
@@ -322,10 +323,10 @@ instance
   where
   wrapFailed = PrtclFailure
 
--- | Calculate the total ada in the chain state
+-- |Calculate the total ada in the chain state
 totalAda :: ChainState crypto -> Coin
```

Comment:
Should this be producing `Value` now?

Reply:
there is now `totalAda` and `totalValue` ,  whichever is relevant is used